lcls_tools/common/measurements/measurement.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+from pydantic import BaseModel, DirectoryPath
+
+
+class Measurement(BaseModel, ABC):
+    name: str
+    save_data: bool = True
+    save_location: Optional[DirectoryPath] = None
+
```

Comment:
should we have metadata placeholders here that all measurements should have? Area? device locations? rmats?

Reply:
Maybe beam path over area...

Reply:
the measurements will contain device objects, which themselves contain the metadata. We can add a property to the measurement object that collects this info for a user if necessary